components/base-rest-api/packages/services/lib/token-swapper-service.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
```

Comment:
This should have some test coverage. I would at least like to see
- golden path
- expired token
- token in the db already revoked
- same token
- no token in database
- for coverage, a test with the revoker service missing from the provider config